src/app/common/hooks/use-bitcoin-contracts.ts

```diff
@@ -66,9 +65,7 @@ export function useBitcoinContracts() {
   const nativeSegwitPrivateKeychain = useNativeSegwitAccountBuilder()?.(currentIndex);
   const currentBitcoinNetwork = useCurrentNetwork();
 
-  async function getBitcoinContractInterface(
-    attestorURLs: string[]
```

Comment:
attestorURLs are not required in this function anymore, because of the new version of @dlc-link/dlc-tools.